src/lib.rs

```diff
@@ -128,9 +120,7 @@ impl EncodingDetector {
             match charset {
                 Err(_) =>
                     panic!("uchardet_get_charset returned invalid value"),
```

Comment:
I know this is my code, but I'd like to change this `panic!` message to something like `"uchardet_get_charset returned a charset name containing invalid characters"`. I can do this when merging unless you want to do it yourself.